modular_skyrat/modules/company_imports/code/armament_datums/sol_defense.dm

```diff
@@ -5,17 +5,38 @@
 // Basic armor vests
 
 /datum/armament_entry/company_import/sol_defense/armor
-	subcategory = "Light Body Armor"
-
-/datum/armament_entry/company_import/sol_defense/armor/slim_vest
-	name = "type I vest - slim"
```

Comment:
as quirky as the new armors are, you shouldn't remove this until there's another option to get standard armor that isn't 'through the crates you can't buy without sec access'

Reply:
If you’re looking for “standard” anything, company imports will soon be not the place to do that once I am done with it, hope this clears things up